client/views/windows/help.tpl

```diff
@@ -37,6 +37,18 @@
 			</p>
 		</div>
 	</div>
+	
+	<div class="help-item">
+		<div class="subject">
+			<kbd class="key-all">Ctrl</kbd><kbd class="key-apple">⌘</kbd> + <kbd>R</kbd>
```

Comment:
Ctrl+R is reload page which is useful, overall I don't think keybind is needed for this at all.